spec.bs

```diff
@@ -4062,10 +4106,14 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
 1. [=list/For each=] |owner| of |owners|:
   1. [=list/For each=] |originalInterestGroup| of the [=user agent=]'s [=interest group set=] whose
     [=interest group/owner=] is |owner| and [=interest group/next update after=] is before
-    the [=current wall time=]:
+    the [=current wall time=] and whose [=interest group/update url=] is not null:
 
     Note: Implementations can consider loading only a portion of these interest groups
     at a time to avoid issuing too many requests at once.
+
+    Note: Implementations can consider aborting all updating if updating has been running for too

```

Comment:
I don't think it's valid to use RFC words like "can" in a non-normative note. I think this sentence should be outside of a note since it directs a browser in a way which impacts web-observable behavior. And mentioning that "some implementations" (i.e., Chromium) have chosen a 10-minute limit for starters, would be good I think, to give others something to go off of.

Reply:
I moved the text outside a Note, and mentioned Chromium's limit. I don't think "can" is listed in https://datatracker.ietf.org/doc/html/rfc2119? Anyhow, let me know what you think of the new wording.

Reply:
Looks like I was wrong about "can". Sorry! This all looks good now though.